README.md

```diff
@@ -1,6 +1,6 @@
 # Project Pythia Portal
 
-![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/NCAR/pythia-portal/deploy-site/main?logo=github&style=for-the-badge)
+![GitHub Workflow Status (branch)](https://img.shields.io/github/actions/workflow/status/NCAR/pythia-portal/ci.yaml?branch=main)
```

Comment:
This is still pointing to the wrong repo.

I think this is the correct badge link:
```suggestion
[![deploy-site](https://github.com/ProjectPythia/projectpythia.github.io/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/ProjectPythia/projectpythia.github.io/actions/workflows/ci.yaml)
```